code/modules/reagents/chemistry/reagents.dm

```diff
@@ -8,6 +8,7 @@
 	var/volume = 0
 	var/metabolization_rate = REAGENTS_METABOLISM
 	var/color = "#000000" // rgb: 0, 0, 0 (does not support alpha channels - yet!)
+	var/complementary_color = "#000000"
```

Comment:
This shouldn't be a reagent level var. It's only blob reagents that use it.

Reply:
I moved the var where it belongs. Fixed.